sdk/communication/communication-administration/test/utils/recordedClient.ts

```diff
@@ -66,45 +66,34 @@ export const environmentSetup: RecorderEnvironmentSetup = {
 };
 
 export function createRecordedCommunicationIdentityClient(
-  context: Context
+  context: Context,
+  withToken: boolean = false
 ): RecordedClient<CommunicationIdentityClient> {
   const recorder = record(context, environmentSetup);
 
-  return {
-    client: new CommunicationIdentityClient(env.COMMUNICATION_CONNECTION_STRING),
-    recorder
-  };
-}
-
-export function createRecordedCommunicationIdentityClientWithToken(
-  context: Context
-): RecordedClient<CommunicationIdentityClient> | undefined {
-  const recorder = record(context, environmentSetup);
-  let credential: TokenCredential;
-
-  if (isPlaybackMode()) {
-    credential = {
-      getToken: async (_scopes) => {
-        return { token: "testToken", expiresOnTimestamp: 11111 };
-      }
-    };
-
+  if (!withToken) {
     return {
-      client: new CommunicationIdentityClient(env.COMMUNICATION_ENDPOINT, credential),
+      client: new CommunicationIdentityClient(env.COMMUNICATION_CONNECTION_STRING),
```

Comment:
I see it does not pass in `credential` to `CommunicationIdentityClient`. why `credential` is not needed now?

Reply:
I guess instead of passing it. 'credential' is now create in line 82 below.

Reply:
yea that is for the ctor that takes a connection string... I should actually wrap this in the try block as well